development/components/form/types-reference/image-preview-type.md

```diff
@@ -0,0 +1,31 @@
+---
+title: ImagePreviewType
+---
+
+# ImagePreviewType
+
+This form type is used to display an image value without providing an interactive input to edit it. It is based on a hidden input so it could be changed programmatically, or be used just to display an image in a form.
```

Comment:
```suggestion
This form type is used to display an image value without providing an interactive input to edit it. It is based on a hidden input so it could be changed programmatically, or used just to display an image in a form.
```